mongodump/mongodump.go

```diff
@@ -38,22 +38,24 @@ type MongoDump struct {
 	InputOptions  *InputOptions
 	OutputOptions *OutputOptions
 
+	SkipUsersAndRoles bool
```

Comment:
I don't know much about Go. Does this need to be initialized to false to keep mongodump/mongorestore working normally or does that happen automatically?